apps/game/locale/fa/LC_MESSAGES/django.po

```diff
@@ -172,8 +172,6 @@ msgstr "دسته بازنده‌ها"
 #: apps/game/templates/scoreboard/friendly_match_scoreboard.html:9
 #: apps/game/templates/scoreboard/group_table.html:10
 #: apps/game/templates/scoreboard/group_table_challenge.html:10
-#, fuzzy
-#| msgid "Score"
```

Comment:
@AmirhosseinS3 Sometimes Django gets confused about translations and puts fuzzy. These are not applied unless someone remove the line `#, fuzzy`

Take care :)